layers/+source-control/git/packages.el

```diff
@@ -29,8 +29,14 @@
         (orgit :requires org)
         smeargle
         transient
+        golden-ratio
```

Comment:
The package name should probably be sorted alphabetically by moving it below `git-timemachine`.